app/templates/play/ladder/ladder.jade

```diff
@@ -1,7 +1,15 @@
 extends /templates/base
+
+mixin showTournamentEnd
```

Comment:
Ah, I meant that if it was an `if / else` for showing the rest of the template, the rest of the template could be a mixin. 

I don't think this is necessary. You can just put the content from this mixin directly here: https://github.com/codecombat/codecombat/pull/5434/files#diff-ff7886813c0f73ab933ea9d0f448270eR11